interfaces/Portalicious/src/app/pages/project/project-team/add-user-button/add-user-button.component.html

```diff
@@ -1,7 +1,94 @@
 <p-button
-  label="Add user to team"
-  i18n-label
-  rounded
+  label="Add team member"
+  i18n-label="@@add-team-member"
+  outlined
   icon="pi pi-plus"
-  (click)="onAdd()"
+  class="text-xs sm:text-base"
+  (click)="formVisible.set(true)"
 />
+
+<app-form-sidebar
+  [(visible)]="formVisible"
+  [formGroup]="formGroup"
+  [mutation]="assignUserMutation"
+  formTitle="Add team member"
+  i18n-formTitle="@@add-team-member"
+>
+  <app-form-field-wrapper
+    label="Choose user"
+    i18n-label
+    [errorMessage]="formFieldErrors()('userValue')"
+    class="mb-3"
+  >
+    <p-dropdown
+      [options]="unassignedUsers()"
+      [optionLabel]="'username'"
+      [optionValue]="'id'"
+      [filter]="true"
```

Comment:
```suggestion
      filter
```

I think? I generally prefer this syntax but maybe just me

Reply:
I prefer the other way. ;)

It emphasizes the 'booleanness' of it, in a way... And is a bit more explicit when reading... But I'm fine with both ways...